docsets/OpenEdge/docset.json

```diff
@@ -1,22 +1,31 @@
-{
-    "name": "OpenEdge",
-    "version": "11.5",
-    "archive": "OpenEdge.tgz",
-    "author": {
-        "name": "Chris Kelleher",
-        "link": "https://twitter.com/chris4gl"
-    },
-    "aliases": ["Progress", 
-                "ABL",
-                "4GL",
-                "WebSpeed",
-                "DBMS"
-               ],
-
-    "specific_versions": [
-        {
-            "version": "11.5",
-            "archive": "versions/11.5/OpenEdge.tgz",
-        }
-    ]
-}
+{
+    "name": "OpenEdge",
+    "version": "11.7/2",
+    "archive": "OpenEdge.11.7.2.docset.tgz",
+    "author": {
+        "name": "Chris Kelleher",
+        "link": "https://twitter.com/chris4gl"
+    },
+    "aliases": ["Progress", 
+                "ABL",
+                "4GL"
+               ],
+    "specific_versions": [
+        {
+            "version": "11.7/2",
+            "archive": "versions/11.7.2/OpenEdge.11.7.2.docset.tgz"
+        }
+        {
+            "version": "11.6/4",
+            "archive": "versions/11.6.4/OpenEdge.11.6.4.docset.tgz"
+        }
+        {
+            "version": "11.5/1",
+            "archive": "versions/11.5.1/OpenEdge.11.5.1.docset.tgz"
+        }

```

Comment:
You're missing a comma at the end of this line.

Reply:
Added missing commas